exercises/sum.cpp

```diff
@@ -6,3 +6,20 @@
   Insert the second number: 2
   Sum: 3
 */
+#include <iostream>
+using namespace std;
+
+int main()
+{
+
+  int first_number, second_number, sum;
+
+  cout << "Enter two integers: ";
+  cin >> first_number >> second_number;
+
+  sum = first_number + second_number;
+
+  cout << first_number << " + " << second_number << " = " << sum;
+
+  return 0;
```

Comment:
Meglio usare una macro per i valori handcoded (non una variabile perchè occupa memora, il simbolo di macro viene sostituito con il valore reale in fase di compilazione, per info: https://www.arduino.cc/reference/en/language/structure/further-syntax/define/):
```diff
+ #define EXIT_SUCCESS 0
....
+ return EXIT_SUCCESS;
- return 0;
```